build/Winfile.props

```diff
@@ -18,6 +18,7 @@
       <RuntimeLibrary Condition="'$(Configuration)'=='Debug'">MultiThreadedDebugDLL</RuntimeLibrary>
       <RuntimeLibrary Condition="'$(Configuration)'=='DebugXPStatic'">MultiThreadedDebug</RuntimeLibrary>
       <EnableEnhancedInstructionSet Condition="'$(Configuration)'=='DebugXPStatic'">NoExtensions</EnableEnhancedInstructionSet>
+      <Optimization Condition="'$(Configuration)'=='DebugXPStatic'">MinSpace</Optimization>
```

Comment:
Do we want this part? Optimization is disabled for debug builds a few lines above.  When I tried to compile this, the compiler errored out due to the incompatibility between this and /RTC1.